### DISCUSSION TITLE
Domain configuration on netbox

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#11126] [General]
Hi all,

I have setup domain for the netbox, say ( netbox-test.com), can anyone tell me where we can configure it ? is it under configuration.py ?
In the allowed hosts, if i just define domain name and the ip address of netbox. then we can always connect to netbox using domain name right. 

In the end i would like to connect ( https://netbox-test.com) like this. I think by default netbox listen on port 8001, should we also configure domain under nginx config file ?

**Comment by tfboy** [comment]
You potentially need to make two changes.

If you want to use SSL (which is recommended), then you'll be using Nginx or Apache as a proxy up front to receive the SSL requests from the clients. You need to have the certificate key and file configured matching the domain name you want to use.

You then need to add the domain name in Netbox's configuration.py file so it will accept the client request. If you have several names / domain names used, then they all need to be listed: `ALLOWED_HOSTS = ['netbox-test.com','netbox-othername.com']`. You can also use a wildcard in configuration.py which makes things easier as you only need to ensure your Nginx / Apache is set up correctly. In which case, it's simply `ALLOWED_HOSTS = ['*']`.
